app/assets/javascripts/discourse/app/widgets/do-not-disturb.js

```diff
@@ -5,7 +5,7 @@ import { iconNode } from "discourse-common/lib/icon-library";
 import showModal from "discourse/lib/show-modal";
 
 export default createWidget("do-not-disturb", {
-  tagName: "div.btn.do-not-disturb-btn",
+  tagName: "li.do-not-disturb-btn",
```

Comment:
The other `li` items in the quick access panel have simple classes (`summary`, `activity` and so on), I would go with `dnd` or `do-not-disturb` here.